drop-in/src/main/java/com/adyen/checkout/dropin/DropInConfiguration.kt

```diff
@@ -271,6 +272,14 @@ class DropInConfiguration : Configuration, Parcelable {
             return this
         }
 
+        /**
+         * Add configuration for Online Banking Poland payment method.
```

Comment:
Poland :)

Reply:
haha a lot of copy pasting as they're the same in more than one thing 😅